### DISCUSSION TITLE
Custom put an image in the builded folder /.output/public and then access the image, why 404 instead of directly display the image?

### DISCUSSION THREAD

**Discussion by wyl20000205** [nuxt/nuxt#25889] [Questions] [answered]
![1](https://github.com/nuxt/nuxt/assets/68469143/4f7ba2e0-ade5-4690-b1f0-39442ff374b4)
![2](https://github.com/nuxt/nuxt/assets/68469143/3b000f82-f846-4fc7-ad2d-1a7981bf496f)
![3](https://github.com/nuxt/nuxt/assets/68469143/5e17cd0a-d94e-4023-879a-83e10170b770)
nuxt version : 3.9.0

**Comment by wyl20000205** [comment, marked as answer]
// This issue has been resolved 😀
// server/routes/images/[name].js
// http://localhost:3000/images/logo.png

import fs from "fs";
import path from 'path'
export default defineEventHandler(async (event) => {
let dest = ''
if (process.env.NODE_ENV == 'production') dest = '.output/public/images';
let filePath = path.join(dest, event.context.params.name);
return sendStream(event, fs.createReadStream(filePath));
});

**Comment by TheAlexLichter** [comment]
Linking https://github.com/nuxt/nuxt/issues/19263 in favor.
